src/lycophron/project.py

```diff
@@ -6,24 +6,33 @@
 # under the terms of the MIT License; see LICENSE file for more details.
 """Lycophron project classes (business logic layer)."""
 
-import csv
-import json
-import os
-import time
-
-from .errors import (
-    DatabaseAlreadyExists,
-    ErrorHandler,
-    InvalidRecordData,
-)
+from .db import LycophronDB
+from .errors import DatabaseAlreadyExists, ErrorHandler
 from .loaders import LoaderFactory
 from .schemas.record import RecordRow
 
 
 class Project:
-    def __init__(self, project_folder=os.getcwd()):
-        self.project_folder = project_folder
+    def __init__(self, db_uri):
         self.errors = []
+        self._db_uri = db_uri
+
+    @property
+    def db(self):
```

Comment:
It's easier to instantiate an app and call:

`app.project.db ...` than importing `db` all the time.